app/_how-tos/kic-vault-hashicorp.md

```diff
@@ -0,0 +1,102 @@
+---
+title: Configure Hashicorp Vault
+description: "Configure Hashicorp Vault with {{ site.kic_product_name }} and the KongVault CRD"
+content_type: how_to
+
+permalink: /kubernetes-ingress-controller/vault/hashicorp/
+breadcrumbs:
+  - /kubernetes-ingress-controller/
+  - /gateway/secrets-management/
+
+min_version:
+  kic: '3.1'
+
+products:
+  - kic
+
+tools:
+  - kic
+
+works_on:
+  - on-prem
+  - konnect
+
+entities: []
+
+tldr:
+  q: How do I configure Hashicorp Vault with {{ site.kic_product_name }}?
+  a: Create a `KongVault` CRD and then use the `vault://` reference in your plugin configuration
+
+prereqs:
+  enterprise: true
+  kubernetes:
+    skip_proxy_ip: true
+  inline: 
+    - title: HashiCorp Vault
+      include_content: prereqs/hashicorp-vault-k8s
+      icon_url: /assets/icons/hashicorp.svg
+
+cleanup:
+  inline:
+    - title: Uninstall KIC from your cluster
+      include_content: cleanup/products/kic
+      icon_url: /assets/icons/kubernetes.svg
+    - title: Uninstall HashiCorp Vault from your cluster
+      content: |
+        ```bash
+        kubectl delete namespace vault
+        ```
+      icon_url: /assets/icons/hashicorp.svg
```

Comment:
This isn't rendering in the output, but from the cleanup file, I think it should be.

Reply:
This is fixed on main